test_scripts/Polices/user_consent_of_Policies/ATF_isAppPermissionsRevoked_true.lua

```diff
@@ -0,0 +1,147 @@
+	end)
+	:Times(AtLeast(1))
+end
+
+function Test:RegisterApp1()
+	commonTestCases:DelayedExp(3000)
+	self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession:StartService(7)
+	:Do(function()
+		local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered")
+		:Do(function(_,data)
+			self.HMIAppID = data.params.application.appID
+		end)
+		self.mobileSession:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS" })
+		self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+	end)
+end
+
+function Test:PTU_revoke_app_group()
+	local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+	EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+	:Do(function()
+		self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+			{
+				requestType = "PROPRIETARY",
+				fileName = "filename"
+			}
+		)
+		EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+		:Do(function()
+			local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest",
+				{
+					fileName = "PolicyTableUpdate",
+					requestType = "PROPRIETARY"
+				}, "files/PTU_AppRevokedGroup.json")
+			local systemRequestId
+			EXPECT_HMICALL("BasicCommunication.SystemRequest")
+			:Do(function(_,data)
+				systemRequestId = data.id
+				self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+				{
+					policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"
+			})
+		local function to_run()
+			self.hmiConnection:SendResponse(systemRequestId,"BasicCommunication.SystemRequest", "SUCCESS", {})
+		end			
+		RUN_AFTER(to_run, 500)
+		end)
+		EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = self.HMIAppID, isAppPermissionsRevoked = true,  appRevokedPermissions = {"Navigation-1"}})
+		:Do(function()
+			local RequestIdListOfPermissions = self.hmiConnection:SendRequest("SDL.GetListOfPermissions", { appID = self.HMIAppID })
+			EXPECT_HMIRESPONSE(RequestIdListOfPermissions)
+			:Do(function()
+				local ReqIDGetUserFriendlyMessage = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"Navigation-1"}})
+				EXPECT_HMIRESPONSE(ReqIDGetUserFriendlyMessage, { result = { code = 0, messages = {{ messageCode = "Navigation-1"}}, method = "SDL.GetUserFriendlyMessage"}})
+			end)
+			self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+			end)
+		end)
+	end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+commonFunctions:userPrint(34, "Test is intended to check that Policies Manager responds isAppPermissionRevoked:true for revoked app permissions")
```

Comment:
rejected. Does it have any impact on template? I guess this is not related to it.